src/dataSources/WikibaseActionApi.js

```diff
@@ -13,26 +13,48 @@ module.exports = class WikibaseActionApi extends RESTDataSource {
     return this.getEntitiesLoader.load(id);
   }
 
+  async searchEntities(query, language) {
+    return await this.getSearchedEntities( {query, language} );
+  }
+
   getEntitiesLoader = new DataLoader(async (ids) => {
     const getEntitiesResponse = await this.get('', {
       action: 'wbgetentities',
       format: 'json',
       ids: ids.join('|')
     });
-
     return ids.map(id => getEntitiesResponse.entities[id]);
   }, {
     maxBatchSize: 50,
   });
 
+  async getSearchedEntities({query, language}) {
+    const searchEntitiesResponse = await this.get('', {
+      action: 'wbsearchentities',
+      format: 'json',
+      search: query,
+      language: language
+    });
+    const ids = [];
+    searchEntitiesResponse.search.forEach( (result) => {
+      ids.push( result.id );
+    });
+    const getEntitiesResponse = await this.get('', {
+      action: 'wbgetentities',
+      format: 'json',
+      ids: ids.join('|')
+    });
+    return ids.map(id => getEntitiesResponse.entities[id]);
```

Comment:
This can reuse the dataloader object and doesn't need to implement its own batching. The following should work as well:
```suggestion
    return searchEntitiesResponse.search.map(({ id }) => this.getEntitiesLoader.load(id));
```